shared/actions/chat2/index.js

```diff
@@ -39,13 +39,45 @@ import flags from '../../util/feature-flags'
 // Ask the service to refresh the inbox
 const inboxRefresh = (
   state: TypedState,
-  action: Chat2Gen.InboxRefreshPayload | Chat2Gen.LeaveConversationPayload
+  action:
```

Comment:
following the better pattern where n-actions being dispatched cause a refresh as a side effect